source/pigeonhole-man/option-x-ext.inc.in

```diff
@@ -0,0 +1,16 @@
+**-x**\ *extensions*
+   Set the available extensions. The parameter is a space-separated list
+   of the active extensions. By prepending the extension identifiers
+   with **+** or **-**, extensions can be included or excluded relative
+   to the configured set of active extensions. If no extensions have a
+   **+** or **-** prefix, only those extensions that are explicitly
+   listed will be enabled. Unknown extensions are ignored and a warning
+   is produced.
+
+   For example **-x** "+editheader -enotify" will enable the editheader
+   extension and disable the enotify extension. The rest of the active
+   extensions depends on the :pigeonhole:ref:`sieve_extensions` and
+   :pigeonhole:ref:`sieve_global_extensions` settings. By default, i.e. when
+   :pigeonhole:ref:`sieve_extensions` and :pigeonhole:ref:`sieve_global_extensions` remain unconfigured,
+   all supported extensions are available, except for deprecated extensions, security sensitive extension,
+   ones that require explicit configuration or those that are still under development.
```

Comment:
Several problems in this sentence; try this:

", except for deprecated extensions, security-sensitive extensions, extensions that require explicit configuration, and those that are still under development."